demos/remote_attestation/init_ra_flow/gen-cert.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Add comments in this file to highlight that user should use this script in a secure environment and carefully protect the generated key, if my understanding is correct.